content/_data/changelogs/weekly.yml

```diff
@@ -5480,6 +5480,78 @@
   # pull: 4080 (PR title: IntelliJ/Java: Duplicate code -- Refactored as DOSToUnixPathHelper.validate)
   # pull: 4354 (PR title: Document workaround for Stapler data binding bug in repeatableProperty)
 
+- version: '2.206'
+  date: 2019-11-24
+  changes:
+
+  # Major RFE
+
+  - type: major rfe
+    message: |-
+      Deprecate the macOS native installer packaging
+    references:
+      - url: https://jenkins.io/blog/2019/11/25/macos-native-installer-deprecation/
+        title: Jenkins macOS native installer deprecation
+    authors:
+      - oleg-nenashev
+
+  # RFEs
+
+  - type: rfe
+    pull: 4362
+    issue: 60189
+    message: |-
+      Open plugin and license links on a separate tab
+    authors:
+      - fqueiruga
+
+  - type: rfe
+    pull: 4363
+    message: |-
+      Set the HttpOnly flag for the page auto-refresh tokens
+    authors:
+      - StefanSpieker
+
+  # Bug fixes
+
+  - type: bug
+    pull: 4348
+    issue: 60092
+    message: |-
+      Avoid exception when views are defined using job DSL.
```

Comment:
This one is not ideal IMHO, but should do the job

Reply:
I'm open to other phrasings.  It is described twice in the changelog, once in the user visible terms (avoid exception) and once in developer terms (behavior has changed for method xYzzY, developers depending on this version or newer must adapt to the new behavior)